DemoQA/Pages/webTablesPage.js

```diff
@@ -0,0 +1,55 @@
+import { Selector, t } from "testcafe";
+import webTablesPage from "../Pages/webTablesPage";
+import UserInfoRow from "./userPage";
+import mainPage from "../Pages/mainPage";
+
+
+class WebTablesPage {
+    constructor () {
+        this.searchBar = Selector(".form-control");
+        this.rowDropDown = Selector(".-pageSizeOptions");
+        this.pageNo = Selector("[type~='number']");
+        this.addBttn = Selector("#addNewRecordButton");
+        this.burgerWebTables = Selector("#item-3");
+        this.rowLine = Selector(".rt-tbody [role='row']");
+        this.menuOptions = Selector("li");
+        this.usersTable = Selector(".ReactTable");
+        this.userRow = Selector(".rt-tr-group");
+    }
+
+    async openMenu(burgerOption) {
+        await t.click(this.menuOptions.withText(burgerOption));
+    }
+
+    async clickAddBttn() {
+        await t.click(this.addBttn);
+    }
+
+    async usersTableList() {
```

Comment:
Please rename this function, because it doesn't answer the 'What does it do?' question. Start function names with a verb.